invenio_rdm_records/cli.py

```diff
@@ -287,23 +293,20 @@ def custom_fields():
 
 
 # helper functions
-def _prepare_mapping(fields_name, available_fields):
+def _prepare_mapping(given_fields_name, available_fields):
     """Prepare ES mapping properties for each field."""
-    available_fields = {field.name: field for field in available_fields}
     fields = []
-    if fields_name:  # create only specified fields
-        for field_name in fields_name:
-            if field_name in available_fields.keys():
-                fields.append(available_fields[field_name])
-            else:
-                click.secho(
```

Comment:
comment: Remove validation and use sets. validation happens before, in the `validate_custom_fields` function. Using sets to avoid double looping on `available_fields.